app/policies/roles/site/institution.json.erb

```diff
@@ -35,15 +77,38 @@
   },
   {
     "key": "technician",
-    "name": "Site <%=  %> Technician",
+    "name": "Site <%= site_name %> Technician",
```

Comment:
They were designed for Lidia and Raul, our technician and supervisor personas: https://docs.google.com/document/d/1BrDnV6FCDhZNaUTkhQXUL1U4nCQJ6D32lORRTnZJDxY/edit?pli=1#heading=h.lrp8tos9y27j
But there is no formal requirement of what they should be able to do if that's what you asked.